docs/docs/deployment/hosting/kubernetes.md

```diff
@@ -425,6 +425,18 @@ The following table lists the configurable parameters of the chart and their def
 
 ---
 
+## Key upgrade notes
+
+- [0.20.0](https://artifacthub.io/packages/helm/flagsmith/flagsmith/0.20.0):
```

Comment:
This link 404s at present, but matches with https://github.com/Flagsmith/flagsmith-charts/pull/123/files#diff-b47ee2ff78b27e1238ffafd64c2155ce5d860904435f39890491876c52cda9f9R5, so will work once that PR is merged.